device_doctor/test/src/android_device_test.dart

```diff
@@ -266,6 +266,43 @@ void main() {
     });
   });
 
+  group('AndroidScreenRotationCheck', () {
+    AndroidDeviceDiscovery deviceDiscovery;
+    MockProcessManager processManager;
+    Process process;
+    List<List<int>> output;
+
+    setUp(() {
+      deviceDiscovery = AndroidDeviceDiscovery('/tmp/output');
```

Comment:
Would this try to write to that location? if it will I'd recommend to use an in memory file instead.

Reply:
Thanks for the comment, used memory file instead.